python/whl.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash -e
+
```

Comment:
Same here, bash can become quickly hard to read...

Reply:
Ack